Library/Homebrew/test/test_bottle_versions.rb

```diff
@@ -75,4 +75,12 @@ def test_ngircd_style
     assert_version_detected '22',
       'ngircd-22.mavericks.bottle.tar.gz'
   end
+
+  def test_apparix_style
+    assert_version_detected '11-062',
+      'apparix-11-062.yosemite.bottle.tar.gz'
+
+    assert_version_detected '11-062',
+      'apparix-11-062..bottle.tar.gz'
```

Comment:
It seems that if t no bottle block presents in the formula, this case will happen.

```
$ brew pry
==> Interactive Homebrew Shell
Example commands available with: brew pry --examples
[1] brew(main)> f=:vim.f;
[2] brew(main)> Bottle.new(f, f.bottle_specification).url
=> "https://homebrew.bintray.com/bottles/vim-7.4.488..bottle.tar.gz"
``